code/modules/mob/living/basic/space_fauna/changeling/flesh_spider.dm

```diff
@@ -33,7 +33,6 @@
 	unsuitable_cold_damage = 4
 	unsuitable_heat_damage = 4
 	combat_mode = TRUE
-	faction = list() // No allies but yourself
```

Comment:
Was there a reason this was removed (is it default behavior now)

Reply:
yes